### PR TITLE
Image refresh for centos-8-stream

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for centos-8-stream
 * [x] image-refresh centos-8-stream
 * https://github.com/cockpit-project/cockpit-machines/pull/473

### CODE REVIEW

**Comment by martinpitt** [comment]
The same `memfd:libffi` SELinux FUBAR as on rhel-8-6.

**Comment by KKoukiou** [comment]
The cockpit-machines issues are solved with: https://github.com/cockpit-project/cockpit-machines/pull/473 Please land these in a lockstep.

**Comment by KKoukiou** [comment]
The cockpit-machines issues are solved with: https://github.com/cockpit-project/cockpit-machines/pull/473 Please land these in a lockstep.

File: machine/machine_core/machine.py [hunk 1]
```diff
@@ -235,6 +235,10 @@ def allowed_messages(self):
             # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1979182
             allowed.append('audit.*denied.*{ getattr }.*comm="systemctl" name="/".*cockpit_ws_t.*')
 
+        if self.image in ['rhel-8-6', 'centos-8-stream']:
```
Comment (martinpitt) [inline machine/machine_core/machine.py:238]: This is already done in #2681, so I suppose this PR needs to be rebased after that lands.

Comment (mvollmer) [reply machine/machine_core/machine.py:238]: Yeah, I just wanted to run the tests in parallel. This will get sorted out during rebasing.

**Comment by mvollmer** [comment]
> The cockpit-machines issues are solved with: [cockpit-project/cockpit-machines#473](https://github.com/cockpit-project/cockpit-machines/pull/473)

That PR seems to have the same failures as here: https://logs.cockpit-project.org/logs/pull-473-20211206-123556-11ab2e24-centos-8-stream-bots-2690/log.html

**Comment by martinpitt** [comment]
This is now only waiting for https://github.com/cockpit-project/cockpit-machines/pull/473 (where the run against this new image is green), so changing "needswork" to "blocked". These two need to land in lockstep.

**Pull request merged**